### PR TITLE
Added cask for elektriktrickquicklook plugin version 1.0.3 (STL and GCode viewer).

### PR DESCRIPTION
**Pull request by BrianGilbert**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [*] `brew cask audit --download {{cask_file}}` is error-free.
- [*] `brew cask style --fix {{cask_file}}` reports no offenses.
- [*] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [*] Named the cask according to the [token reference].
- [*] `brew cask install {{cask_file}}` worked successfully.
- [*] `brew cask uninstall {{cask_file}}` worked successfully.
- [*] Checked there are no [open pull requests] for the same cask.
- [*] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/elektriktrickquicklook.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'elektriktrickquicklook' do
+  version '1.0.3'
+  sha256 :no_check
```
Comment (miccal) [inline Casks/elektriktrickquicklook.rb]: If the `version` is set, the `sha256` also needs to be set.

**Comment by BrianGilbert** [comment]
@fanquake Added sha256, cheers.

File: Casks/elektriktrickquicklook.rb (hunk 1 above)
Comment (BrianGilbert) [inline Casks/elektriktrickquicklook.rb]: Have now added sha256 value

**Review by vitorgalvao — APPROVED** [review]

**Comment by BrianGilbert** [comment]
@vitorgalvao thanks for removing that bit.

**Pull request merged**